INSTALL.md

```diff
@@ -159,8 +159,9 @@ If the compiling process stops with a missing dependency like: "... libswscale/s
 And install the package that provide the missing file.
 Start make again. Repeat if nessary until all dependencies are installed.
 
+For Fedora: modify the  INCLUDE in /home/qTox/Makefile (or wherever you cloned the repo) ADDING -isystem/usr/include/ffpmeg OR  cd %{Git_Dir}/qTox/src; wget https://github.com/tux3/qTox/pull/1881 && patch -p1 ## to avoid   fatal error  compiling  libavcodec/avcodec.h  
```

Comment:
No.

First of all, your instruction is broken.

Secondly, even if it wasn't broken it wouldn't have place in INSTALL.md, since, since that's just a workaround…